packages/backend/src/services/UnfurlService/UnfurlService.ts

```diff
@@ -813,7 +813,6 @@ export class UnfurlService extends BaseService {
                         });
                     }
 
-                    Sentry.captureException(e);
```

Comment:
No need to do this, the screenshot/unfurl is already being caught in SchedulerTaskTracer.